packages/builder/src/builders/application/index.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  BuilderContext,
+  createBuilder,
+  fromAsyncIterable,
+} from "@angular-devkit/architect";
+import {
+  ApplicationBuilderOptions,
+  buildApplication,
+} from "@angular-devkit/build-angular";
+import { dotenvRun } from "@dotenv-run/esbuild";
+import { DotenvRunOptions } from "@dotenv-run/webpack";
+import { from, switchMap, mergeMap, map } from "rxjs";
+import { indexHtml } from "../../utils/esbuild-index-html";
+import { getProjectCwd } from "../../utils/project";
+import { NgxEnvSchema } from "../ngx-env/ngx-env-schema";
+
+export const buildWithPlugin = (
+  options: ApplicationBuilderOptions & NgxEnvSchema,
+  context: BuilderContext
+) => {
+  const dotEnvOptions: DotenvRunOptions = {
+    ...options.ngxEnv,
+    appEnv: "NG_APP_ENV",
```

Comment:
isn't his overriding a custom prefix from the user?

Reply:
that's what you did in plugin.ts using webpack.
```export function plugin(options: DotenvRunOptions, ssr = false) {
  const _options = { ...options, appEnv: "NG_APP_ENV" };
  const dotEnvPlugin = new DotenvRunPlugin(_options, ssr);
  const raw = dotEnvPlugin.raw;```

Reply:
Oh yeah my bad I've just read the docs 😅

> Note that NG_APP_ENV remains available even if you define a custom prefix not matching NG_APP.